### PR TITLE
Add network performance page

### PR DESCRIPTION
**Pull request by hsanjuan**

### CODE REVIEW

File: docs/about-filecoin/network-performance.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+3. Deal shows up on-chain: Once the data has successfully transferred, the miner must complete a critical step before the storage deal can show up on-chain. This step is referred to as “PreCommit”, and takes around ~Z time to complete for a minimum viable mining machine for one sector.
+
+4. Sector sealing: Once the deal shows up on-chain, the miner must still complete [generating a Proof-of-Replication and sealing the sector](https://spec.filecoin.io/#systems__filecoin_mining__sector__adding_storage). This process is currently estimated to take ~1.5hours for a 32GB sector on a machine that meets these [minimum hardware requirements for mining](../mine/hardware-requirements.md#general-hardware-requirements).
+
+For most storage clients, the storage performance that matters is the time it takes from deal acceptance to deal appearance on-chain. This is the sum of steps (1) to (3) above. Based on current high-level benchmarks, these steps are estimated to take around ~5-10 minutes for a 1MiB file.
```
Comment (hsanjuan) [inline docs/about-filecoin/network-performance.md:29]: It should say hours?

File: docs/about-filecoin/network-performance.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- **Retrieval after unsealing**: Because of the Filecoin protocol’s design, storage miners are essentially cryptographically guaranteed to store client data in its sealed format. Therefore, if the storage miner doesn’t have an unsealed copy of the data stored, they will have to unseal the sealed data first (i.e. decoding the encoded data) and then serve it back up to the requester, i.e. the retrieval client.
+
+In both methods, the data retrieval process after a retrieval deal is accepted includes:
+
+1. **Funding the payment channel for retrieval**: Similar to the storage deal payment channel funding above, except for the purpose of data retrieval. The timing estimates for payment channel creation and funding are roughly the same as noted above.
+
+2. **Unsealing (if needed)**: The miner unseals (decodes) the data so it can be read by the data requester. Sealing and unsealing are symmetric processes, which means they take roughly the same amount of time in either direction. Thus, the unsealing step is estimated to take around as long as the sealing step listed above, or around ~1.5hours for a 32GiB sector on a machine running minimum hardware requirements.
```
Comment (hsanjuan) [inline docs/about-filecoin/network-performance.md:42]: Verify value.

Comment (magik6k) [reply docs/about-filecoin/network-performance.md:42]: 2-3h with unreleased optimizations

File: docs/about-filecoin/network-performance.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+# {{ $frontmatter.title }}
+
+The Filecoin Network is a decentralized storage market and network that provides data persistence via a decentralized protocol and publicly verifiable storage proofs on a blockchain. Current Filecoin network performance is primarily determined by security parameters and Filecoin's [proof constructions](https://spec.filecoin.io/#algorithms__pos).
+
+It is difficult to provide highly reliable benchmarks for Filecoin network performance today due to the network being extremely early stage and therefore, somewhat volatile, and because Filecoin is a distributed system. However, as you begin interacting with Filecoin, you can use these heuristics to understand general Filecoin network performance and how it fits with your use case.
+
+## Financial transfers
+
+A message that requires [transferring FIL](../get-started/lotus/send-and-receive-fil.md#sending-fil) is often extremely fast, and will take on average ~1 blocktime (or around 30 seconds) to succeed.
```
[the remark this answers is not in the document]
Comment (magik6k) [reply docs/about-filecoin/network-performance.md:15]: 120 blocks is a value I've heard from red team as 'ok for big transfers', 900 is finality / ultra-conservative.

For smaller value transfers probably ask the red team

File: docs/about-filecoin/network-performance.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Data storage
+
+The Filecoin [data storage protocol](../store/lotus/store-data.md) has a few key components once a deal is proposed and accepted:
+
+1. Funding the payment channel: This process takes ~3-5 minutes if this is the first payment channel between a client and a miner. If this is the second or later payment channel between a given client and miner, the payment channel funding process takes ~X minutes.
```
[the remark this answers is not in the document]
Comment (magik6k) [reply docs/about-filecoin/network-performance.md:21]: Payment channels are only used for retrieval

[the remark this answers is not in the document]
Comment (magik6k) [reply docs/about-filecoin/network-performance.md:21]: (This is likely adding funds to the storage market actor)

File: docs/about-filecoin/network-performance.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+2. Data transfer: This portion of the deal flow involves the client’s node sending the relevant pieces of data to the mining node. Assuming all goes well -- no broken connections -- this process can transfer at a rate of ~Y MB/s. This data transfer rate varies widely, depending on the bandwidth of both the client and the miner, the geographic distance that the data has to travel, and more. Please take all of these estimates as heuristics, not hard-and-fast rules or guaranteed SLA!
```
[the remark this answers is not in the document]
Comment (magik6k) [reply docs/about-filecoin/network-performance.md:23]: Yes, it really depends

File: docs/about-filecoin/network-performance.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+3. Deal shows up on-chain: Once the data has successfully transferred, the miner must complete a critical step before the storage deal can show up on-chain. This step is referred to as “PreCommit”, and takes around ~Z time to complete for a minimum viable mining machine for one sector.
```
[the remark this answers is not in the document]
Comment (magik6k) [reply docs/about-filecoin/network-performance.md:25]: Precommit is a part of the sealing process. Deals are published to the chain before the sectors even start sealing

[the remark this answers is not in the document]
Comment (hsanjuan) [reply docs/about-filecoin/network-performance.md:25]: @magik6k how long does it take to seal "normally" ?

**Pull request merged**